README.md

```diff
@@ -1,2 +1,23 @@
 # lobster-ice
 STEP 2020 Capstone Project :) 
+
+### Step 1: Start Server
+- create a file in this directory name'.env' for credential setup. (reach out for details)
+- cd server
```

Comment:
I think '$ cd server' is clearer to indicate this is a terminal command.

Reply:
okay!